radio/src/translations/bl_translations.h

```diff
@@ -82,11 +82,51 @@
     #define TR_BL_CURRENT_FW               "Aktualni firmware:"
 
     #define TR_BL_SELECT_KEY               BL_SELECT_KEY " pro vybrani souboru"
-    #define TR_BL_FLASH_KEY                "Drzet dlouze " BL_SELECT_KEY " pro nahrani"
+    #define TR_BL_FLASH_KEY                "Drzet dlouho " BL_SELECT_KEY " pro nahrani"
```

Comment:
I don't think a PR to add a new language should be changing an existing language translation,